spec/secrets.yml

```diff
@@ -52,9 +52,15 @@ components:
           required: true
           schema:
             $ref: 'openapi.yml#/components/schemas/ResourceID'
+        - name: "expirations"
+          in: "query"
+          description: "Tells the server to reset the variables expiration date"
+          required: false
+          schema:
+            type: string
```

Comment:
Seeing as `expirations` is only ever used as an empty string, I see this being more accurate as a boolean

Reply:
The reason I didn't use a boolean is because the actual value of the parameter doesn't matter. So whether you passed `true` or `false` to the kwarg it would always perform the `expirations` action. Therefore I think it would be misleading to use a bool. Most people would expect that if they passed `false` for `expirations` it would not perform that action, however it always would.

Reply:
Ohh that's fair. Didn't realize this in my `roles` tests.